src/routes/controllers/productsControllers/getProductByID.js

```diff
@@ -0,0 +1,17 @@
+import ProductModel from '../../../models/product';
+
+const getProductByID = async (req, res) => {
+  try {
+    const productID = req.params.productID;
+    const product = await ProductModel.findById(productID);
+    if (product) {
+      res.status(200).json(product);
+    } else {
+      res.status(200).json('Product not exist');
```

Comment:
* when product doesnt exists status code should be 404. Also, when you send error just send it as object with key msg as you do with Internal server error
* this whole if else can be simplified with truthy value of product. something similar to 
<img width="593" alt="Screen Shot 2021-10-31 at 12 37 41 AM" src="https://user-images.githubusercontent.com/54900075/139567944-e63c81ff-8f56-47ee-88b7-7095f9f21211.png">